common/v2/features/Settings/components/GeneralSettings.tsx

```diff
@@ -94,31 +102,48 @@ export default class GeneralSettings extends React.Component<SettingsProps> {
     }
   };
 
+  public renderTooltipContent = () => <div>Test</div>;
+
   public render() {
     const { globalSettings } = this.props;
     return (
-      <DashboardPanel heading="General Settings">
+      <DashboardPanel heading={translateRaw('SETTINGS_GENERAL_LABEL')}>
         <Divider />
         <SettingsField>
-          <SettingsLabel>Account Settings</SettingsLabel>
+          <SettingsLabel>
+            {translateRaw('SETTINGS_HANDLING_LABEL')}
+            <Tooltip tooltip={<span>{translateRaw('SETTINGS_TOOLTIP')}</span>}>
+              {props => (
+                <div {...props}>
+                  <SettingsTooltipIcon icon="shape" />
+                </div>
+              )}
+            </Tooltip>
```

Comment:
1. Tooltip hover is still jumpy. I was under the impression that Povio had corrected this.

Reply:
They did but it's a seperate change from the ui library. I believe we just need to publish a new version. It's separate from core though.

Reply:
Then can we please publish the new version and update the package in core?

Reply:
I'll get that published